keyboards/handwired/scottokeebs/scottosplit/keymaps/default/config.h

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2022 Joe Scotto
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#define DEBOUNCE 5
+
+// Define options
+#define IGNORE_MOD_TAP_INTERRUPT
```

Comment:
```suggestion
```
This directive is deprecated and will trigger a build error. Consider removing it proactively from all your existing pull requests.

Reply:
@filterpaper Will do right now

Reply:
@filterpaper Done, also updated Pro Micro boards to use `"development_board": "promicro",`

Is there something equivalent for RP2040 boards?

Reply:
You can peruse the schema files for the list of supported RP2040 boards (and other json directives):
https://github.com/qmk/qmk_firmware/blob/master/data/schemas/keyboard.jsonschema#L36